src/app/component-library/catalog/asset-detail-dialog/asset-detail-dialog.component.html

```diff
@@ -187,8 +187,14 @@
         <button
           *ngIf="!data.asset.isOwnConnector"
           mat-raised-button
+          matTooltip="{{
+            canNegotiate
+              ? 'Start a negotiation process'
```

Comment:
You can default to null or an empty string, I believe. Normal buttons do not need a tooltip, I'd say. Also prefer to use `[matTooltip]="canNegotiate ? null : 'My message'"` over `{{ }}` inside attributes which is more of a "last resort" string concatenation